converters/fromZigbee.js

```diff
@@ -4136,6 +4136,43 @@ const converters = {
             }
         },
     },
+    evanell_thermostat: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandDataResponse', 'commandDataReport'],
+        convert: (model, msg, publish, options, meta) => {
+            const dp = msg.data.dp; // First we get the data point ID
+            const value = tuya.getDataValue(msg.data.datatype, msg.data.data);
+
+            switch (dp) {
+            case tuya.dataPoints.evanellChildLock:
+                return {child_lock: value ? 'LOCK' : 'UNLOCK'};
+            case tuya.dataPoints.evanellBattery:
+                return {battery: value};
+            case tuya.dataPoints.evanellHeatingSetpoint:
+                // value is already converted to a number in JS, and we deduced that it needs to be divided by 10
+                return {current_heating_setpoint: (value / 10).toFixed(1)};
+            case tuya.dataPoints.evanellLocalTemp:
+                return {local_temperature: parseFloat((value / 10).toFixed(1))};
```

Comment:
Don't use to fixed here and 2 lines above, it will convert the number to  a string